caddy/caddy_test.go

```diff
@@ -305,3 +308,153 @@ func TestPHPServerDirectiveDisableFileServer(t *testing.T) {
 	tester.AssertGetResponse("http://localhost:9080", http.StatusOK, "I am by birth a Genevese (i not set)")
 	tester.AssertGetResponse("http://localhost:9080/hello.txt", http.StatusNotFound, "Not found")
 }
+
+func TestMetrics(t *testing.T) {
+	var wg sync.WaitGroup
+	tester := caddytest.NewTester(t)
+	tester.InitServer(`
+	{
+		skip_install_trust
+		admin localhost:2999
+		http_port 9080
+		https_port 9443
+
+		frankenphp
+	}
+
+	localhost:9080 {
+		route {
+			php {
+				root ../testdata
+			}
+		}
+	}
+	`, "caddyfile")
+
+	// Make some requests
+	for i := 0; i < 10; i++ {
+		wg.Add(1)
+		go func(i int) {
+			tester.AssertGetResponse(fmt.Sprintf("http://localhost:9080/index.php?i=%d", i), http.StatusOK, fmt.Sprintf("I am by birth a Genevese (%d)", i))
+			wg.Done()
+		}(i)
+	}
+	wg.Wait()
+
+	// Fetch metrics
+	resp, err := http.Get("http://localhost:2999/metrics")
+	if err != nil {
```

Comment:
nit: you can use `require.NoError()` and other assertions provided by [testify](https://github.com/stretchr/testify) because we already require and use this tool.